### DISCUSSION TITLE
Installazione su altervista

### DISCUSSION THREAD

**Discussion by alessandroandrei** [iisgiua/giuaschool#227] [Richieste di aiuto]
Buonasera, non riesco in alcun modo ad installare il registro sul mio hosting Altervista. 

Dopo aver seguito la procedura d'installazione fornita dallo sviluppatore, vado per provare a installare (tramite browser Chrome) ma mi viene riportato un errore. 

Come posso risolvere questo problema?  

Grazie in anticipo per il supporto e l'aiuto

**Comment by manuelbarberi** [comment]
Scrivimi su supporto@giuademo.x10.mx .... ti aiuterò io ... inoltrami le credenziali altervista

**Reply by alessandroandrei** [reply]
Grazie, email appena inviata

**Reply by manuelbarberi** [reply]
Fatto

**Comment by ghost** [comment]
@alessandroandrei Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
